linters/markdown-link-check/plugin.yaml

```diff
@@ -0,0 +1,30 @@
+version: 0.1
+tools:
+  definitions:
+    - name: markdown-link-check
+      runtime: node
+      package: markdown-link-check
+      shims: [markdown-link-check]
+      known_good_version: 3.11.2
+lint:
+  definitions:
+    - name: markdown-link-check
+      files: [markdown]
+      tools: [markdown-link-check]
+      commands:
+        - name: lint
+          # Custom parser type defined in the trunk cli to handle markdownlint's JSON output.
+          output: sarif
+          run: markdown-link-check -q "${target}"
+          success_codes: [0, 1]
+          cache_results: false
+          read_output_from: stdout
+          parser:
+            runtime: python
+            run: python3 ${plugin}/linters/markdown-link-check/parse.py
+      # do not suggest yet / markdown-link-check doesn't provide line/col info yet and a good report
+      # suggest_if: files_present
```

Comment:
nit: for now, add `suggest_if: never` to be explicit

Reply:
done